packages/ui/src/views/repo/pull-request/components/pull-request-diff-viewer.tsx

```diff
@@ -210,113 +267,286 @@ const PullRequestDiffViewer = ({
+    ({ onClose, side, lineNumber }) => {
+      const sideKey = side === SplitSide.old ? 'oldFile' : 'newFile'
+      const commentText = getNewCommentValue(side, lineNumber)
+
+      return (
+        <div className="flex w-full flex-col border px-[4px] py-[8px]">
+          <PullRequestCommentBox
+            isEditMode
+            inReplyMode
+            onSaveComment={() => {
+              onClose()
+              if (commentText.trim() && handleSaveComment) {
+                handleSaveComment(commentText.trim(), undefined, {
+                  line_end: lineNumber,
+                  line_end_new: sideKey === 'newFile',
+                  line_start: lineNumber,
+                  line_start_new: sideKey === 'newFile',
+                  path: fileName
+                })
+              }
+              setNewCommentValue(side, lineNumber, '')
+            }}
+            currentUser={currentUser}
+            onCancelClick={() => {
+              onClose()
+              setNewCommentValue(side, lineNumber, '')
+            }}
+            comment={commentText}
+            setComment={value => setNewCommentValue(side, lineNumber, value)}
+          />
+        </div>
+      )
+    },
+    [handleSaveComment, fileName, newComments, currentUser]
+  )
+
+  const renderExtendLine = useCallback<NonNullable<DiffViewProps<Thread[]>['renderExtendLine']>>(
+    ({ data: threads }) => {
+      if (!threads) return <></>
+
+      return (
+        <div className="border bg-background rounded w-">
+          {threads.map(thread => {
+            const parent = thread.parent
+            const componentId = `activity-code-${parent?.id}`
+            const parentIdAttr = `comment-${parent?.id}`
+            const replies = thread.replies
+            const parentInitials = getInitials(parent.author ?? '', 2)
             return (
-              <>
-                <div className="bg-background/50 px-6 py-[6px]">
-                  {info?.data.map((item, index) => {
-                    return (
-                      <Card key={index} className="rounded-md bg-transparent">
-                        <div className="flex flex-col p-4">
-                          <div className="flex items-center space-x-2 font-sans">
-                            <Avatar className="size-5">
-                              <AvatarFallback className="p-1 text-center text-xs">
-                                {getInitials(currentUser ?? '', 2)}
-                              </AvatarFallback>
-                            </Avatar>
-                            <Text color="primary">{currentUser ?? ''} </Text>
+              <PullRequestTimelineItem
+                key={parent.id}
+                id={parentIdAttr}
+                parentCommentId={parent.id}
+                handleSaveComment={handleSaveComment}
+                isLast={true}
+                contentClassName="px-4 py-2 w-[calc(100%-38px)]"
```

Comment:
Can we move this number `38px` as a separate css variable? It'll help with readability and purpose of it.